public/index.html

```diff
@@ -16,20 +16,6 @@
     crossorigin=""></script>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
-
-  <script type="text/javascript">
-    if( document.referrer.indexOf('map.nachbarschaftsmarktplatz') == -1){
-      $(window).on('load', function () {
-      $('#Modal1').modal('show');
-      });
-    } else {
-      $(window).on('load', function () {
```

Comment:
The 'else' part is still necessary if we want the shop registration to pop up when visiting the website.